torch/_export/serde/serialize.py

```diff
@@ -153,7 +153,6 @@ def _reverse_map(d: Dict[Any, Enum]):
     operator.sub,
     operator.floordiv,
     operator.mod,
-    torch.sym_sqrt,
```

Comment:
Here I just removed duplicated item in the set. There is another `torch.sym_sqrt` on the line 161.